sale_coupon_free_shipping_default_product/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Sale coupon free shipping default product",
+    "summary": "avoids creating multiple free shipping products",
+    "version": "14.0.1.0.0",
+    "category": "Hidden/Tools",
+    "author": "Ooops, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-promotion",
+    "license": "LGPL-3",
+    "installable": True,
+    "depends": ["sale_management", "stock", "coupon"],
```

Comment:
Hi, thank you for your contribution @Chionne27.

Small change, i think there are unnecessary module dependencies. Could you please check?

I think that the only one needed is `coupon`
```suggestion
    "depends": ["coupon"],
```

Reply:
Oh thank you for the suggestion. Let me know if everything is ok